### PR TITLE
V2Wizard: Oscap step

### PR DESCRIPTION
**Pull request by mgold1234**
this commit adds the Oscap component to V2Wizrad.

### CODE REVIEW

File: src/store/imageBuilderApi.ts [hunk 1]
```diff
@@ -721,6 +721,7 @@ export type PackagesResponse = {
   data: Package[];
 };
 export type DistributionProfileItem =
+  | "None"
```
Comment (lavocatt) [inline src/store/imageBuilderApi.ts:724]: Hey, just a quick comment on this one:

One thing to remember is that the api files shouldn't get updated manually. These files are generated by the `npm run api` command. The command is loading the openapi yaml file for the image builder api and generating the types for us. If you change manually the file, the next time someone will update the yaml file, your changes will be erased and lost.

And the other thing to remember is that the openapi yaml file in the frontend *has to* be in sync with image builder's file that lives in the image builder repository too.

If you feel like a None option is necessary, the workflow is the following:
* You need to first issue a PR on image builder to have its API changed.
* Once it's merged, import the new yaml file into the frontend repository.
* Run the `npm run api` command
* And issue a PR on the frontend :slightly_smiling_face:

Comment (lavocatt) [reply src/store/imageBuilderApi.ts:724]: BTW, I don't think adding a `None` option to the image builder api is a good idea. In the API perspective, not sending a profile means `None` already.

**Comment by lavocatt** [comment]
I've been running the code locally using the mock service workers. There's an issue with in the OScap step, where on `msw` a profile is selected by default 
image
it's not doing the same without the msw, I think it's worth investigating why.

**Comment by mgold1234** [comment]
I run this with npm run stage-beta:experimental.
@lucasgarfield should we be able to run it with msw as well?

File: src/store/imageBuilderApi.ts (hunk 1 above)
Comment (mgold1234) [reply src/store/imageBuilderApi.ts]: I remove it from the imageBuilderApi. thank you

**Comment by mgold1234** [comment]
I rebase the code and dill with a lot of conflict, so closing this pr and opend new one. 
https://github.com/RedHatInsights/image-builder-frontend/pull/1559

**Pull request closed without merging**